app/interactors/album.js

```diff
@@ -22,4 +22,16 @@ exports.purchaseAlbum = (user, album) => {
 
   });
 
+};
+
+exports.getAlbumsByUser = (id) => {
+
+  return new Promise((resolve, reject) => {
```

Comment:
You can simple do the following:
`return Album.findAll({ where: { userId: id }})`
thats because the findAll method returns a promise, so you can simply return that promise instead of creating a new one.